CC/Sounder/ClientRadioSet.cc

```diff
@@ -28,69 +28,49 @@ static void freeRadios(std::vector<Radio*>& radios)
 ClientRadioSet::ClientRadioSet(Config* cfg)
     : _cfg(cfg)
 {
-    Radio* new_radio;
-    bool has_runtime_error;
+    size_t num_radios = _cfg->num_cl_sdrs();
 
     //load channels
     auto channels = Utils::strToChannels(_cfg->cl_channel());
     radios.clear();
-    radios.reserve(_cfg->num_cl_sdrs());
```

Comment:
This statement doesn't hurt anything.  If the .resize gets called alot on a particular vector, this will provide a high water mark that might be helpful with memory allocations.  Probably fine to remove in this condition